translations/Arabic/special-terms.md

```diff
@@ -0,0 +1,178 @@
+# Terms
+
+
+- Message: رسالة
+
+Direct translation in all contexts
```

Comment:
One issue to fix for rendering is adding whitespace in everything that should be indented with the `-` - could you fix that? In the GitHub preview nothing is intended.